bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/RowMerger.java

```diff
@@ -364,6 +364,7 @@ private FlatRow buildRow() {
   private ByteString lastCompletedRowKey = null;
   private RowInProgress rowInProgress;
   private boolean complete;
+  private int rowCountInLastMessage = -1;
```

Comment:
should this be volatile?

Reply:
RowMerger should only be called from one thread.  We should have docs for that :)